shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalSecrets.java

```diff
@@ -33,26 +30,17 @@ public String getKey(String secretName) throws SecretRetrievalException {
         String key = "";
 
         try {
-            key = readSecretFromFileSystem(secretName);
-        } catch (SecretRetrievalException exception) {
-            logger.logWarning(
-                    "Not finding the " + secretName + " on the filesystem, searching in resources");
             key = readSecretFromResources(secretName);
+        } catch (SecretRetrievalException exception) {
+            logger.logWarning("Not finding the " + secretName + " in resources");
```

Comment:
We probably want to just throw a runtime exception and not start the app if a secret is missing instead of throwing another of the same type of exception

Reply:
The try catch was nuked in the base PR.